corehq/apps/experiments/tests/test_experiment.py

```diff
@@ -0,0 +1,209 @@
+import gc
+from contextlib import contextmanager
+from time import sleep
+from unittest.mock import patch
+
+import pytest
+from unmagic import fixture
+
+from corehq.util.metrics.tests.utils import capture_metrics
+from corehq.util.test_utils import capture_log_output
+
+from .. import Experiment
+
+experiment = Experiment(
+    campaign='test',
+    old_args={'x': 2},
+    new_args={'x': 4},
+)
+
+
+def test_experiment():
+    @experiment
+    def func(a, *, x):
+        return a + x
+
+    with capture_metrics() as metrics, capture_log_output("notify") as log:
+        assert func(1) == 3
+    assert log.get_output() == "func(1): 3 != 5\n"
```

Comment:
I can't wrap my head around this assertion - Can you explain why we will get this output in the logs?

Reply:
`experiment` is configured with two different values for `x` (2 and 4). When the experiment is run the result of the old code path will be 1 + 2 = 3 and the new code path will be  1 + 4 = 5. When the old and new code path results cause `is_equal` to return false, the difference is logged to Sentry (which is done by logging to the "notify" logger):

https://github.com/dimagi/commcare-hq/blob/73dcecd94cf0f37f706b9cb96f77883033ec5875/corehq/apps/experiments/experiment.py#L134-L144